app/services/authentication.py

```diff
@@ -44,7 +76,13 @@ def send_beta_backlog_email(email_address):
     send_beta_backlog_notification_email.queue(email_address)
 
 
+def send_registration_email(email_address):
+    token = encode_token(email_address, "email-confirm-salt")
+    confirm_url = url_for("account.confirm", token=token, _external=True)
+    send_confirm_email.queue(email_address, confirm_url)
```

Comment:
Actually - this is the function that it calls.  It might be a little indirect.  I'll move the little logic in these function into the UserNotificationService object.